src/components/KorneevDS/react-tic-tac-toe/App.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import game_styles from './TicTacToe.css';
```

Comment:
```suggestion
import game_styles from './TicTacToe.module.css';
```
Иначе названия классов так и не поменяются